docs/user_guide.md

```diff
@@ -21,34 +21,34 @@
     - Assign `Bank` the `Custody`, `Distribution`, and `Settlement` roles
     - Assign `Exchange` the `Exchange` role
     - Assign `Ccp` the `Clearing` role
-9. Request services from specific parties: (then click Next)
+8. Request services from specific parties: (then click Next)
```

Comment:
thank you for the number changes!

Reply:
I did go a little overboard with that. Prettier can auto-format the line numbers in markdown files, which is how I got by before